mozdownload/scraper.py

```diff
@@ -697,14 +697,19 @@ def get_build_info(self):
         self.builds = parser.entries
         self.build_index = len(parser.entries) - 1
 
-        if self.build_number and \
-                ('build%s' % self.build_number) in self.builds:
+        if self.build_number is None:
+            self.logger.info('Selected build: %s' %
+                             (parser.entries[self.build_index]))
+            return
+
+        if ('build%s' % self.build_number) in self.builds:
```

Comment:
Feel free to remove the early return, and make this an `elif` instead.

Reply:
For me early return makes code easier to read. It separates code blocks for handling:
- case when build_number is not passed and
- case when build_number is passed and it is needed to do additional checks.